com.unity.render-pipelines.universal/ShaderLibrary/Debug/DebugViewEnums.cs

```diff
@@ -3,110 +3,232 @@
+    /// </summary>
     [GenerateHLSL]
     public enum DebugVertexAttributeMode
     {
+        /// <summary>No vertex attribute debug.</summary>
         None,
+        /// <summary>Display texture coordinate 0.</summary>
         Texcoord0,
+        /// <summary>Display texture coordinate 1.</summary>
         Texcoord1,
+        /// <summary>Display texture coordinate 2.</summary>
         Texcoord2,
+        /// <summary>Display texture coordinate 3.</summary>
         Texcoord3,
+        /// <summary>Display vertex color.</summary>
         Color,
+        /// <summary>Display tangent.</summary>
         Tangent,
+        /// <summary>Display normal.</summary>
         Normal,
     }
+
+    /// <summary>
+    /// Debug mode for validating out-of-range values of different material channels.
+    /// </summary>
+    [GenerateHLSL]
+    public enum DebugMaterialValidationMode
+    {
+        /// <summary>No material debug validation override.</summary>
+        None,
+        /// <summary>Validate albedo values according to validation settings.</summary>
+        Albedo,
+        /// <summary>Validate metallic values according to validation settings.</summary>
+        Metallic
+    }
+
     #endregion
 
     #region Rendering Settings
+
+    /// <summary>
+    /// Debug mode for displaying intermediate render targets.
+    /// </summary>
     [GenerateHLSL]
     public enum DebugFullScreenMode
     {
+        /// <summary>No intermediate render target displayed.</summary>
         None,
+        /// <summary>Display depth buffer contents.</summary>
         Depth,
+
+        /// <summary>Display the shadow map from additional lights.</summary>
         AdditionalLightsShadowMap,
+        /// <summary>Display the main shadow map.</summary>
         MainLightShadowMap,
     }
 
+    /// <summary>
+    /// Debug mode that overrides how the renderer behaves.
+    /// </summary>
     [GenerateHLSL]
     public enum DebugSceneOverrideMode
     {
+        /// <summary>No debug override.</summary>
         None,
+        /// <summary>Visualize overdraw by drawing geometry using a semitransparent material. Areas that look opaque contain more overdraw.</summary>
         Overdraw,
+        /// <summary>Render using wireframe only.</summary>
         Wireframe,
+        /// <summary>Render using a constant fill color and wireframe.</summary>
         SolidWireframe,
+        /// <summary>Render shaded geometry in addition to wireframe.</summary>
         ShadedWireframe,
     }
 
+    /// <summary>
+    /// Debug modes for texture mipmaps.
+    /// </summary>
     [GenerateHLSL]
     public enum DebugMipInfoMode
     {
+        /// <summary>No mipmap debug.</summary>
         None,
+        /// <summary>Display the mipmap level sampled.</summary>
         Level,
+        /// <summary>Display the amount of mip levels available.</summary>
         Count,
+        /// <summary>Display the mip ratio.</summary>
+        Ratio
     }
```

Comment:
Should we expose this since we don't support Mip level debug?

Reply:
The mode was not removed completely, just the UI was disabled. I can remove this and all related shader code.